src/pages/publication/DescriptionPanel.tsx

```diff
@@ -79,7 +78,6 @@ const DescriptionPanel: FC<DescriptionPanelProps> = ({ goToNextTab, savePublicat
     <TabPanel ariaLabel="description" goToNextTab={goToNextTab} onClickSave={validateAndSave}>
       <MuiPickersUtilsProvider utils={DateFnsUtils}>
         <FormCard>
-          <FormCardHeading>{t('heading.description')}</FormCardHeading>
```

Comment:
Nåja. Vi snakket om å ha den der selv om den var redundant

Reply:
fra kortet: `Fjern “Beskrivelse“ som overskrift (det står i arkfanen)`

Reply:
arkfanen burde sikkert vært hatt h1

Reply:
synes også vi skal beholde denne inntil de har fått på plass konsistent design gjennom alle fanene. For en som bruker en screenreader er det også mye enklere å skjønne konteksten av siden når det står som en overskrift vs at man må tabbe seg opp til fanene for å finne det ut.

Reply:
oki, tar det tilbake. uansett smal sak å endre igjen senere